docs/src/pages/components/date-range-picker/CustomDateRangePickerDay.js

```diff
@@ -1,53 +1,42 @@
 import * as React from 'react';
-import { createTheme } from '@material-ui/core/styles';
-import { makeStyles } from '@material-ui/styles';
+import { styled } from '@material-ui/core/styles';
 import Box from '@material-ui/core/Box';
 import TextField from '@material-ui/core/TextField';
 import AdapterDateFns from '@material-ui/lab/AdapterDateFns';
 import LocalizationProvider from '@material-ui/lab/LocalizationProvider';
 
 import StaticDateRangePicker from '@material-ui/lab/StaticDateRangePicker';
-import DateRangePickerDay from '@material-ui/lab/DateRangePickerDay';
-import clsx from 'clsx';
+import MuiDateRangePickerDay from '@material-ui/lab/DateRangePickerDay';
 
-const defaultTheme = createTheme();
-
-const useStyles = makeStyles(
-  (theme) => ({
-    highlight: {
+const DateRangePickerDay = styled(MuiDateRangePickerDay)(
+  ({ theme, isHighlighting, isStartOfHighlighting, isEndOfHighlighting }) => ({
+    ...(isHighlighting && {
       borderRadius: 0,
       backgroundColor: theme.palette.primary.main,
       color: theme.palette.common.white,
       '&:hover, &:focus': {
         backgroundColor: theme.palette.primary.dark,
       },
-    },
-    firstHighlight: {
+    }),
+    ...(isStartOfHighlighting && {
       borderTopLeftRadius: '50%',
       borderBottomLeftRadius: '50%',
-    },
-    endHighlight: {
+    }),
+    ...(isEndOfHighlighting && {
       borderTopRightRadius: '50%',
       borderBottomRightRadius: '50%',
-    },
+    }),
   }),
-  { defaultTheme },
 );
 
 export default function CustomDateRangePickerDay() {
-  const classes = useStyles();
   const [value, setValue] = React.useState([null, null]);
 
   const renderWeekPickerDay = (date, dateRangePickerDayProps) => {
     return (
-      <DateRangePickerDay
-        {...dateRangePickerDayProps}
-        className={clsx(dateRangePickerDayProps.className, {
-          [classes.firstHighlight]: dateRangePickerDayProps.isStartOfHighlighting,
-          [classes.endHighlight]: dateRangePickerDayProps.isEndOfHighlighting,
-          [classes.highlight]: dateRangePickerDayProps.isHighlighting,
-        })}
-      />
+      <React.Fragment>
```

Comment:
seems like `React.Fragment` is not needed.

Reply:
Correct, I needed to add @ts-ignore directive temporary, I should remove them now.